node/core/backing/src/lib.rs

```diff
@@ -650,11 +650,19 @@ impl CandidateBackingJob {
 			// spawn background task.
 			let bg = async move {
 				if let Err(e) = validate_and_make_available(params).await {
-					tracing::error!(
-						target: LOG_TARGET,
-						"Failed to validate and make available: {:?}",
-						e
-					);
+					if let Error::BackgroundValidationMpsc(error) = e {
+						tracing::debug!(
+							target: LOG_TARGET,
+							?error,
+							"Mpsc background validation mpsc died during validation- leaf no longer active?"
+						);
+					} else {
+						tracing::debug!(
```

Comment:
any reason to downgrade this to debug?

Reply:
Yep: https://matrix.to/#/!JZkrzbabXFGsaweNTt:matrix.parity.io/$R_p9_Cna8w4I1335zW0VX6jO8OBhOScg2mHhNdskkTo?via=matrix.parity.io&via=web3.foundation

I think that the closed mpsc is really an expected event, in case the leaf gets deactivated before the validation was able to finish. Which I guess can easily happen in practice (network delays and such) and should not trigger an error and alerting people. That's at least my understanding, if I am wrong - let's fix the bug instead :sweat_smile:

Reply:
I guess it should not happen too often though ... if we want to keep an eye on that, a metric and alerts when it happens too often, in contrast to a higher logging level, would probably be more effective?

Reply:
oh - I just realized what you meant - no that still should be error, that was a copy paste error - thank you!